### PR TITLE
Background pack for ReactOS.

### PR DESCRIPTION
**Pull request by Xcallono**
Also a hopeful test for Dropbox. Might've been too unprofessional here. Please edit as needed.

### CODE REVIEW

File: xbgpack1.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+ Version = 1
+ License = CC0
+ Description = Super Mario 64 based backgrounds for ReactOS. 1024x1024 each.
+ Size = 7.36 MB
+ Category = 15
+ URLSite = https://xcallono.cf
+ URLDownload = https://www.dropbox.com/s/ow467t0piyzfdgi/Mario%2064%20Backgrounds.zip?dl=1
+ SHA1 = 13385B2AD4794855CCC4AED0189FE8E943BE23C8
+ CDPath = none
+ SizeBytes = 7725056
```
Comment (katahiromz) [inline xbgpack1.txt]:
```suggestion
 SizeBytes = 7721027
```
Not size in disk but real size. Add new line to the end.

File: xbgpack1.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+[Section]
+ Name = Super Mario 64 Backgrounds!
+ Version = 1
+ License = CC0
+ Description = Super Mario 64 based backgrounds for ReactOS. 1024x1024 each.
+ Size = 7.36 MB
+ Category = 15
```
Comment (katahiromz) [inline xbgpack1.txt]:
```suggestion
 Category = 3
```
I think this category is `ENUM_CAT_GRAPHICS`.

Comment (shytzedaka) [reply xbgpack1.txt]: I don't know if that makes sense. The graphics category is used for programs that have to do with graphics and image editing. putting backgrounds/wallpapers there would seem a bit out of place, don't you think?

Comment (katahiromz) [reply xbgpack1.txt]: OK. I reject my suggestion.

File: xbgpack1.txt (hunk 1 above)
Comment (katahiromz) [reply xbgpack1.txt]: back

Comment (Xcallono) [reply xbgpack1.txt]: Ah, I thought the size for the disk would work better. My apologies, I'll fix that ASAP.

File: xbgpack1.txt [hunk 3]
```diff
@@ -0,0 +1,12 @@
+[Section]
+ Name = Super Mario 64 Backgrounds!
```
Comment (katahiromz) [inline xbgpack1.txt]: Don't add space at the beginning of each line.

File: xbgpack1.txt [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+ Version = 1
+ License = CC0
+ Description = Super Mario 64 based backgrounds for ReactOS. 1024x1024 each.
+ Size = 7.36 MB
+ Category = 15
+ URLSite = https://xcallono.cf
+ URLDownload = https://www.dropbox.com/s/ow467t0piyzfdgi/Mario%2064%20Backgrounds.zip?dl=1
+ SHA1 = 13385B2AD4794855CCC4AED0189FE8E943BE23C8
+ CDPath = none
+ SizeBytes = 7721027
```
Comment (katahiromz) [inline xbgpack1.txt]: Append a new line at the end of file.

**Comment by katahiromz** [comment]
Why closed?

**Pull request closed without merging**